examples/max-supply-precision/index.mjs

```diff
@@ -14,8 +15,14 @@ const assertEq = (a, e) => {
   }
 };
 
+{ // sync indexer with latest blocks, so it can observe token metadata
+  const ctc = acc.contract(backend);
+  await ctc.p.Alice({});
+  await ctc.e.sync.next();
+}
```

Comment:
This program could also work by adding a way to explicitly wait on the indexer. The current `wait` function just probes the node